index.html

```diff
@@ -296,13 +296,13 @@ <h3>
           <li>The <dfn id='dom-PerformanceNavigationTiming-workerStart'><code>
             workerStart</code></dfn> attribute MUST return the time immediately
             before the user agent <a href=
-            "https://www.w3.org/TR/service-workers/#service-worker-concept">ran
+            "https://www.w3.org/TR/service-workers/#run-service-worker">ran
```

Comment:
use `data-cite="service-workers-1#run-service-worker"` here instead of href.

Reply:
ack